paths/paths_windows.go

```diff
@@ -99,5 +99,5 @@ func ensureStateDirPerms(dirPath string) error {
 // LegacyStateFilePath returns the legacy path to the state file when it was stored under the
 // current user's %LocalAppData%.
 func LegacyStateFilePath() string {
-	return filepath.Join(os.Getenv("LocalAppData"), "Tailscale", "server-state.conf")
+	return filepath.Join(os.Getenv("LocalAppData"), "Tailscale", "server-state-trilogy.conf")
```

Comment:
This is a backwards compatibility feature for very old versions of the Windows client. You shouldn't update this filename, those old versions did not have "trilogy" in the filename.